pyfolio/timeseries.py

```diff
@@ -1024,6 +1024,28 @@ def gen_drawdown_table(returns, top=10):
     return df_drawdowns
 
 
+def rolling_volatility(returns, rolling_vol_window):
+    """
+    Determines the rolling volatility of a strategy.
+
+    Parameters
+    ----------
+    returns : pd.Series
+        Daily returns of the strategy, noncumulative.
+         - See full explanation in tears.create_full_tear_sheet.
+    rolling_vol_window : int
+        Length of rolling window, in days, over which to compute.
+
+    Returns
+    -------
+    pd.Series
+        Rolling volatility.
+    """
+
+    return returns.rolling(rolling_vol_window).std() \
```

Comment:
I think the one from empyrical does not work, since it returns a float and we need a pd.Series instead.